libraries/DateTime/contracts/ExampleContract.aes

```diff
@@ -0,0 +1,131 @@
+include "DateTimeLibrary.aes"
+contract ExampleContract =
+
+  public function get_year(timestamp : int) : int = 
+    let year : int = Date.get_year(timestamp)
+    year
+
+  public function get_month(timestamp : int) : int = 
+    let month : int = Date.get_month(timestamp)
+    month
+    
+  public function get_day(timestamp : int) : int = 
+    let day : int = Date.get_day(timestamp)
+    day
+    
+  public function get_hour(timestamp : int) : int = 
+    let hour : int = Date.get_hour(timestamp)
+    hour
+    
+  public function get_minute(timestamp : int) : int = 
+    let minute : int = Date.get_minute(timestamp)
+    minute
+
+  public function get_second(timestamp : int) : int = 
+    let second : int = Date.get_second(timestamp)
+    second
+
+  public function get_weekday(timestamp : int) : int = 
+    let second : int = Date.get_weekday(timestamp)
+    second
+
+  public function is_leap_year(year : int) : bool = 
+    let is_leap_year : bool = Date.is_leap_year(year)
+    is_leap_year
+
+  public function to_timestamp(year : int,
+                               month : int,
+                               day : int,
+                               hour : int,
+                               minute : int,
+                               second : int) : int = 
+    let timestamp : int = Date.to_timestamp(year,month,day,hour,minute,second)
+    timestamp
+
+  public function add_years(timestamp : int, years : int) : int = 
+    let new_tampstamp = Date.add_years(timestamp, years)
```

Comment:
why `new`?

Reply:
`tampstamp` typo

Reply:
can directly return (true often times in this contract)

Reply:
done